windows/utils/win_get_set_disk_online.yml

```diff
@@ -38,7 +38,7 @@
     win_powershell_cmd: "{{ win_cmd_online_disk }}"
   when:
     - win_online_disk_ops is defined
-    - win_online_disk_ops | lower == 'set'
+    - win_online_disk_ops == 'set'
```

Comment:
It seems nowhere to set default value for var "win_online_disk_ops" as the parameter's description in line 10.

Reply:
It's not set explicitly but the logic is the default. Changed to be more clear. Thanks.